### ISSUE TITLE
More BrokenPipeError

### ISSUE THREAD

**Issue by kalefranz** [conda/conda#6582]
```
error: BrokenPipeError(32, 'Broken pipe')
command: /home/ubuntu/anaconda3/envs/tensorflow_p36/bin/conda install -p /home/ubuntu/anaconda3/envs/tensorflow_p36 tensorflow-gpu -y
user_agent: conda/4.4.4 requests/2.18.4 CPython/3.6.3 Linux/4.4.0-1044-aws ubuntu/16.04 glibc/2.23
_messageid: -9223372036845075603
_messagetime: 1514219033248 / 2017-12-25 10:23:53

Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/exceptions.py", line 724, in __call__
    return func(*args, **kwargs)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/cli/main.py", line 78, in _main
    exit_code = do_call(args, p)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/cli/conda_argparse.py", line 75, in do_call
    exit_code = getattr(module, func_name)(args, parser)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/cli/main_install.py", line 11, in execute
    install(args, parser, 'install')
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/cli/install.py", line 255, in install
    handle_txn(progressive_fetch_extract, unlink_link_transaction, prefix, args, newenv)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/cli/install.py", line 281, in handle_txn
    progressive_fetch_extract.execute()
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/core/package_cache.py", line 575, in execute
    exc = self._execute_actions(prec_or_spec, prec_actions)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/core/package_cache.py", line 590, in _execute_actions
    progress_bar = ProgressBar(desc, not context.verbosity and not context.quiet, context.json)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/common/io.py", line 365, in __init__
    self.pbar = tqdm(desc=description, bar_format=bar_format, ascii=True, total=1)
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/_vendor/tqdm/_tqdm.py", line 767, in __init__
    self.postfix, unit_divisor))
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/_vendor/tqdm/_tqdm.py", line 201, in print_status
    fp_write('\r' + s + (' ' * max(last_len[0] - len_s, 0)))
  File "/home/ubuntu/anaconda3/lib/python3.6/site-packages/conda/_vendor/tqdm/_tqdm.py", line 195, in fp_write
    fp_flush()
BrokenPipeError: [Errno 32] Broken pipe

```

**Comment by kalefranz** [comment]
```
error: BrokenPipeError(32, 'Broken pipe')
command: /Users/Rex/anaconda3/bin/conda install --yes --json --force-pscheck --prefix /Users/Rex/anaconda3/envs/test spyder==3.2.5
user_agent: conda/4.4.4 requests/2.18.4 CPython/3.6.3 Darwin/17.3.0 OSX/10.13.2
_messageid: -9223372036843575583
_messagetime: 1514436380331 / 2017-12-27 22:46:20

Traceback (most recent call last):
  File "/Users/Rex/anaconda3/lib/python3.6/site-packages/conda/exceptions.py", line 724, in __call__
    return func(*args, **kwargs)
  File "/Users/Rex/anaconda3/lib/python3.6/site-packages/conda/cli/main.py", line 78, in _main
    exit_code = do_call(args, p)
  File "/Users/Rex/anaconda3/lib/python3.6/site-packages/conda/cli/conda_argparse.py", line 75, in do_call
    exit_code = getattr(module, func_name)(args, parser)
  File "/Users/Rex/anaconda3/lib/python3.6/site-packages/conda/cli/main_install.py", line 11, in execute
    install(args, parser, 'install')
  File "/Users/Rex/anaconda3/lib/python3.6/site-packages/conda/cli/install.py", line 255, in install
    handle_txn(progressive_fetch_extract, unlink_link_transaction, prefix, args, newenv)
  File "/Users/Rex/anaconda3/lib/python3.6/site-packages/conda/cli/install.py", line 296, in handle_txn
    common.stdout_json_success(prefix=prefix, actions=actions)
  File "/Users/Rex/anaconda3/lib/python3.6/site-packages/conda/cli/common.py", line 168, in stdout_json_success
    stdout_json(result)
  File "/Users/Rex/anaconda3/lib/python3.6/site-packages/conda/cli/common.py", line 161, in stdout_json
    json.dump(d, sys.stdout, indent=2, sort_keys=True, cls=EntityEncoder)
  File "/Users/Rex/anaconda3/lib/python3.6/json/__init__.py", line 180, in dump
    fp.write(chunk)
BrokenPipeError: [Errno 32] Broken pipe
```

**Comment by kalefranz** [comment]
resolved with #6616
